dash/include/dash/Execution.h

```diff
@@ -0,0 +1,69 @@
+#ifndef DASH__EXECUTION_H__INCLUDED
+#define DASH__EXECUTION_H__INCLUDED
+
+#include <type_traits>
+
+
+namespace dash {
+
+/**
+ * Execution policy type trait.
+ */
+template<class T>
+struct is_execution_policy
+: public std::integral_constant<bool, false>
+{ };
+
+namespace execution {
+
+  /**
+   * Sequential execution policy.
+   */
+  class sequenced_policy { };
+
+  /**
+   * Parallel execution policy.
+   */
+  class parallel_policy { };
+
+  /**
+   * Parallel non-sequential execution policy.
+   */
+  class parallel_unsequenced_policy { };
```

Comment:
This is well described on [cppreference](http://en.cppreference.com/w/cpp/algorithm/execution_policy_tag_t).